operatorhub/openshift/config.yaml

```diff
@@ -209,9 +209,9 @@ image-substitutions:
       - deploymentName: openshift-pipelines-operator
         containerName: openshift-pipelines-operator
         envKeys:
-          - IMAGE_PAC_CONTROLLER
-          - IMAGE_PAC_WEBHOOK
-          - IMAGE_PAC_WATCHER
+          - IMAGE_PAC_PAC_CONTROLLER
+          - IMAGE_PAC_PAC_WEBHOOK
+          - IMAGE_PAC_PAC_WATCHER
```

Comment:
I see prefix is here https://github.com/tektoncd/operator/blob/main/pkg/reconciler/common/transformers.go#L43

How does `PAC_CONTROLLER`, `PAC_WEBHOOK`, `PAC_WATCHER` matches ? is it in CSV?

Reply:
@savitaashture yeah, in the payload, the name is `pac-controller`, so this works with `IMAGE_PAC` + `PAC_…`

Reply:
(tried it out locally with https://github.com/openshift-pipelines/pipelines-as-code/pull/735)